Application/src/view/visuals/component/Message.java

```diff
@@ -128,8 +134,8 @@ else if (keepAnimating && node1.getX() == node2.getX()) { //Sending a self refer
                     selfCallCounter =1;
             }
         }
-      
-      
+
```

Comment:
?

Reply:
@timjon idk, has to do with spaces vs tabs thingy, couldn't undo this tbh

Reply:
OK

Reply:
The what now?

Reply:
wat

Reply:
What do you mean with tabs vs space thingy? i have no idea what you mean

Reply:
apparently my IDE replaced 2 empty line that has a 4 spaces or tab in them with just empty lines, idk why git showed this and i couldn't reverse it as you can see in commit history @Sebastian-Fransson